docs/querying/sql-functions.md

```diff
@@ -1227,35 +1269,146 @@ Returns a number for each output row of a groupBy query, indicating whether the
 
 ## HLL_SKETCH_ESTIMATE
 
-`HLL_SKETCH_ESTIMATE(expr, [round])`
+Returns the distinct count estimate from an HLL sketch. The `expr` argument must return an HLL sketch. The optional `round` boolean argument rounds the estimate if set to true, with a default of false.
 
-**Function type:** [Scalar, sketch](sql-scalar.md#sketch-functions)
+* **Syntax:** `HLL_SKETCH_ESTIMATE(expr, [round])`
+* **Function type:** Scalar, sketch
 
-Returns the distinct count estimate from an HLL sketch.
+
+<details><summary>Example</summary>
+
+The following example estimates the number of unique tail numbers in the `flight-carriers` datasource.
+
+```sql
+SELECT
+  HLL_SKETCH_ESTIMATE(DS_HLL("Tail_Number"), False) AS "estimate"
+FROM "flight-carriers"
+```
+
+Returns the following:
+
+| `estimate` | 
+| -- |
+| `4,685.8815405960595` |
+
+</details>
+
+[Learn more](sql-scalar.md#sketch-functions)
 
 ## HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS
 
-`HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr, [numStdDev])`
+Returns the distinct count estimate and error bounds from an HLL sketch. The `expr` argument must return an HLL sketch. The optional round `numStdDev` argument rounds the estimate if set to true, with a default of false.
 
-**Function type:** [Scalar, sketch](sql-scalar.md#sketch-functions)
+* **Syntax:** `HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr, [numStdDev])`
+* **Function type:** Scalar, sketch
 
-Returns the distinct count estimate and error bounds from an HLL sketch.
+<details><summary>Example</summary>
+
+The following example estimates the number of unique tail numbers in the `flight-carriers` datasource with error bounds at plus or minus one standard deviation.
+
+```sql
+SELECT
+  HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(DS_HLL("Tail_Number"), 1) AS "estimate_with_errors"
+FROM "flight-carriers"
+```
+
+Returns the following:
+
+| `estimate_with_errors` |
+| -- |
+| `[4685.8815405960595, 4536.952802775876, 4839.925808688653]` |
+
+
+</details>
+
+[Learn more](sql-scalar.md#sketch-functions)
 
 ## HLL_SKETCH_TO_STRING
 
-`HLL_SKETCH_TO_STRING(expr)`
+Returns a human-readable string representation of an HLL sketch for debugging. The `expr` argument must return an HLL sketch.
```

Comment:
I don't think we need to include this sentence for every HLL function:

> The `expr` argument must return an HLL sketch.

To me, it's clear from the description that the function works on HLL sketches. Also `expr` is not an argument. It's a parameter and it doesn't return anything.

Reply:
Removing the sentence and will start distinguishing parameter and argument